pkg/kubelet/container/runtime.go

```diff
@@ -46,12 +46,10 @@ type Runtime interface {
 	GetPods(all bool) ([]*Pod, error)
 	// RunPod starts all the containers of a pod within a namespace.
 	RunPod(*api.Pod, map[string]volume.Volume) error
+	// Syncs the running pod into the desired pod.
+	SyncPod(pod *api.Pod, runningPod Pod, podStatus api.PodStatus) error
```

Comment:
@yujuhong I mean since it's there, let's use it for now. I totally agree we should refactor that part :)